.github/workflows/main.yml

```diff
@@ -49,6 +49,7 @@ jobs:
           python -m cibuildwheel --output-dir dist
 
       - uses: actions/upload-artifact@v2
+        if: ${{ matrix.python-version }} == '3.9'
```

Comment:
Thats the wrong place.

Reply:
:-) Yeah thanks... somehow my text editor must have mysteriously scrolled up...